src/Interpreters/Streaming/JoinStreamDescription.h

```diff
@@ -21,8 +21,8 @@ struct JoinStreamDescription
         Block input_header_,
         DataStreamSemanticEx data_stream_semantic_,
         UInt64 keep_versions_,
-        Int64 latency_threshold_,
-        Int64 quiesce_threshold_ms_)
+        Int64 latency_threshold_ = 0,
```

Comment:
Let's remove this default settings to expose compile issue everywhere. It is a good thing to do this check.